lnpeer/peer.go

```diff
@@ -12,10 +12,17 @@ import (
 // Peer is an interface which represents the remote lightning node inside our
 // system.
 type Peer interface {
-	// SendMessage sends a variadic number of message to remote peer. The
-	// first argument denotes if the method should block until the message
-	// has been sent to the remote peer.
-	SendMessage(sync bool, msg ...lnwire.Message) error
+	// SendMessage sends a variadic number of high-priority message to
+	// remote peer.  The first argument denotes if the method should block
+	// until the messages have been sent to the remote peer or an error is
+	// returned, otherwise it returns immediately after queuing.
+	SendMessage(sync bool, msgs ...lnwire.Message) error
```

Comment:
Any reason we don't just include a bool to denote the lazy message rather than having two separate methods?

Reply:
was thinking it'd be nice to have them separate for clarity and to prevent callers from accidentally swapping `sync` and `lazy` bools. certain subsytems may only need access to one or the other as well